test/auth/facebook.test.js

```diff
@@ -64,14 +64,10 @@ describe('facebook login test', () => {
     sinon.assert.calledOnce(json);
   });
   it('register a new user sucessfully', async () => {
-    const res = { status() {}, json() {} };
-    const status = sinon.stub(res, 'status').returnsThis();
-    const json = sinon.stub(res, 'json').returnsThis();
+    const res = { status() {}, json() {}, redirect() {} };
```

Comment:
Where is the assertion here?

Reply:
That was an oversight. It's been handled now. Thank you.